torch/fx/passes/net_min_base.py

```diff
@@ -21,22 +22,23 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+@compatibility(is_backward_compatible=False)
```

Comment:
cc @842974287 FYI I'm conservatively setting backward-compatibility for the APIs in the file to `False`. Please feel free to flip this to `True` if/when you feel comfortable providing stability guarantees for the API.